ern-container-gen-android/src/AndroidGenerator.ts

```diff
@@ -707,7 +691,49 @@ You should replace "${annotationProcessorPrefix}:${dependency}" with "annotation
           'lib/src/main/java/com/walmartlabs/ern/container/plugins',
         );
         shell.cp(pathToPluginConfigHook, pathToCopyPluginConfigHookTo);
+
+        this.updatePluginsToAndroidxImports(
+          rnVersion,
+          plugin,
+          pathToCopyPluginConfigHookTo,
+        );
+      }
+    }
+  }
+
+  /**
+   * Modify Plugins to support Androidx
+   * eg: import android.support.annotation.nonNull => import androidx.annotation.nonNull
+   */
+  public updatePluginsToAndroidxImports(
+    rnVersion: string,
+    plugin: PackagePath,
+    outDir: string,
+  ) {
+    if (semver.gte(rnVersion, '0.60.0')) {
+      const filesWithSupportLib: string[] = [];
+      shell.pushd(outDir);
+      shell
+        .ls('-R', '.')
+        .filter((file) => file.match(/\.(java|kt)$/))
+        .forEach((file) => {
+          if (shell.grep('android.support', file).trim().length !== 0) {
+            filesWithSupportLib.push(file);
+          }
+        });
+
+      if (filesWithSupportLib.length !== 0) {
+        log.info(
+          `${plugin.name} contains source files with references to the Android Support Library (android.support.*)`,
+        );
+        filesWithSupportLib.forEach((file) => {
+          shell.sed('-i', 'android.support', 'androidx', file);
+        });
+        log.info(
+          `${filesWithSupportLib.length} files successfully converted to use AndroidX (androidx.*)`,
+        );
       }
+      shell.popd();
```

Comment:
This statement is currently _outside_ the if block, and we do _not_ want this to be executed if rn version is < 0.60 - Please move into if block (or, after addressing suggested refactoring, the if block will go away completely).